src/ValueObject/MethodName.php

```diff
@@ -72,4 +72,9 @@ final class MethodName
      * @var string
      */
     public const INVOKE = '__invoke';
+
+    /**
+     * @var string
+     */
+    public const SET = '__set';
```

Comment:
@TomasVotruba it seems cause PHPStan notice as duplicated already registered, I use existing `__SET` const in next commit at PR https://github.com/rectorphp/rector-src/pull/1318 https://github.com/rectorphp/rector-src/pull/1318/commits/276c9c20dbdd5d7c3a444ec2815e514eab3e89b5

Reply:
Ah, thank you :+1: